tests/testthat/test-template.R

```diff
@@ -46,6 +46,27 @@ test_that("UTF-8 characters in templates", {
   expect_identical(charToRaw(text), as.raw(c(0xc3, 0xbf)))
 })
 
+test_that("UTF-8 characters in template head but not body", {
+  # On Windows, a string with "中文" will automatically be marked as UTF-8.
+  ui <- tagList(
+    tags$head(tags$script("alert('中文')")),
```

Comment:
FWIW, I checked the package on r-hub (Windows 2008 Server, R 3.3.2) and there were no errors. I assume that the locale isn't set to Chinese there.
https://builder.r-hub.io/status/htmltools_0.3.6.tar.gz-93b5e051f7594422861bf773c32de71f